### PR TITLE
Add `Runners::ConfigGenerator` class

### LINKED ISSUE #1303: Provide some API tasks as gem
## What

The Runners gem provides some API tasks such as:

- Validate `sider.yml`
- Generate `sider.yml`

## Why

> Validate `sider.yml`

Currently, we have to start analysis via all the tools to just validate `sider.yml`. This is useless.
If there would be an API to validate `sider.yml`, we would not need such a useless analysis.

>  Generate `sider.yml`

If users want to customize analysis, an API to generate `sider.yml` would help them start such a task, e.g. `rubocop --init` or `eslint --init`.

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

This change adds a new class `Runners::ConfigGenerator` that generates an example of a `sider.yml` file.

Usage:

```ruby
Runners::ConfigGenerator.new.generate(tools: [:eslint, :rubocop])
```

> Link related issues or pull requests.

Close #1303

> Check the following if needed.

- [x] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

**Comment by ybiquitous** [comment]
@yykamei Could you give me any feedback for this change, please?

File: lib/runners/config_generator.yml.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This is a configuration file to customize code analysis by Sider.
+#
+# For more information, see the documentation:
+# https://help.sider.review/getting-started/custom-configuration
+
+<%- unless tools.empty? -%>
+# Customize each tools. If analyses fail, try adjusting each option referencing the following example.
```
Comment (yykamei) [inline lib/runners/config_generator.yml.erb:7]:
```suggestion
# Customize each tool. If analyses fail, try adjusting each option referencing the following example.
```

File: test/config_generator_test.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  end
+
+  private
+
+  def assert_yaml(expected_filename, tools:, comment_out_lines:)
+    actual = Runners::ConfigGenerator.new.generate(tools: tools)
+
+    assert_equal data(expected_filename).read, actual
+
+    # Comment out
```
Comment (yykamei) [inline test/config_generator_test.rb:35]: Maybe, this action is uncommenting.

```suggestion
    # Uncomment
```

**Review by yykamei — APPROVED** [review]
LGTM

**Comment by ybiquitous** [comment]
@yykamei Thank you. I've fixed them via 9627c419, so I'll merge this when the CI passes. 👍

**Pull request merged**